### ISSUE TITLE
Another attempt at updating to newest discord

### ISSUE THREAD

**Issue by Chaprnks** [maddinat0r/samp-discord-connector#130]
I had a previous issue relating to the callback breaking changes, which I got solved.

My issue now is that after load, the discord bot does not connect. I noticed a release mentioning an environment variable, where exactly is that stored? I checked pawn-env and it mentioned ".env" file, but that didnt work. Basically I want to know where to set "SAMP_DISCORD_BOT_TOKEN " seeing as the server.cfg method isnt supported anymore.

I'd also like to mention, that the current functions and settings has a successful bot connection via 0.2.4; so its just a code breaking issue getting the connection to load in the newest version id discord connector

**Comment by Chaprnks** [comment]
I'd also like to mention, that the current functions and settings has a successful bot connection via 0.2.4; so its just a code breaking issue getting the connection to load in the newest version id discord connector

**Comment by maddinat0r** [comment]
Not sure if you closed it on purpose, but I'll answer anyway.

You can specify the bot token two ways. The first method is through the server.cfg config file; you'll need to add this line
```
discord_bot_token yourbottoken
```
The second method is through the environment variable `SAMP_DISCORD_BOT_TOKEN`.
If you use both, the environment variable (as long as it's not empty) will take precedence.

The current (R41-4) MySQL plugin and Discord connector plugin won't conflict, but you'll have two log-core libraries in your server root folder: `log-core.dll` (the "old" log-core, shipped with that MySQL plugin version) and `log-core2.dll` (the recent log-core, shipped with the Discord connector v0.3 and higher). There will eventually be a MySQL plugin update that'll use the new log-core.

**Comment by Chaprnks** [comment]
I'm sorry, I should have replied before closing the ticket. I figured out what it was. One of the discord staff members in my community kicked the bot I created from the server & didn't inform me. After re-inviting him to the channel, the latest version of discord worked flawlessly.

---

I also figured out the environment variable. I was confused after seeing the pawn_env plugins on github, thinking that what you were referring to. After realizing you were talking about linux core enviornment variables (ex: ubuntu, export %%) I also was able to get the variables to load successfully.

---

Thank you for your assistance & quick response!
